website/content/en/docs/deployment/cognito-rds-s3/guide-terraform.md

```diff
@@ -123,6 +123,9 @@ export TF_VAR_minio_aws_secret_access_key=
 ### (Optional) Configure Culling for Notebooks
 Enable culling for notebooks by following the [instructions]({{< ref "/docs/deployment/configure-notebook-culling.md#" >}}) in configure culling for notebooks guide.
 
+### Multi-User Profiles
```

Comment:
Creating Profiles?

A default profile named `kubeflow-user-example-com` for email `user@example.com` has been configured with this deployment.

If you are using IRSA as `PIPELINE_S3_CREDENTIAL_OPTION`, any additional profiles that you create will also need to be configured with IRSA and S3 Bucket access. Follow the [pipeline profiles]({{< ref "/docs/deployment/pipeline-profiles.md" >}}) for instructions on how to create additional profiles.

If you are not using this feature, you can create a profile by just specifying email address of the user.